### DISCUSSION TITLE
Customize Collapse for single item

### DISCUSSION THREAD

**Discussion by some-user123** [element-plus/element-plus#13031] [Ideas]
I'm wanting to collapse/expand a certain part of my page. Collapse can do that, but seems to have multiple items in focus. I only have a single item.

It would nice if I could further customize this component, e.g. I'm thinking of:
- Removing the borders
- Customizing the header/title further, e.g. putting the icon to the left, changing the icon

E.g. like markdown does renders it:

<details>
  <summary>Click me</summary>
  
  Content
</details>

**Comment by kaine0923** [comment]
I think you could use the `el-collapse-transition` to customize one

**Reply by ryuhangyeong** [reply]
+1 https://element-plus.org/en-US/guide/transitions.html#collapse
